data/patch.json

```diff
@@ -298,6 +298,18 @@
       "comment": "extend by non-standard width keywords https://developer.mozilla.org/en-US/docs/Web/CSS/width",
       "syntax": "<length> | <percentage> | auto | max-content | min-content | fit-content | fill-available | <-non-standard-width>"
     },
+    "motion": { 
+      "syntax": null 
```

Comment:
add `"comment": "remove when mdn/data released a version after 2.0.1"` for all added properties